certified-connectors/EBMS/apiProperties.json

```diff
@@ -0,0 +1,102 @@
+      "username": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Username",
+          "description": "EBMS Username",
+          "tooltip": "Provide the Username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Password",
+          "description": "EBMS Password",
+          "tooltip": "Provide the Password",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "serialNumber": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Serial Number",
+          "description": "EBMS Serial Number",
+          "tooltip": "Provide the Serial Number",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "companyId": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Company Id",
+          "description": "EBMS Company Id",
+          "tooltip": "Provide the Company Id",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#3776AF",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Set url",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://ecc@connectionParameters('SerialNumber').servicebus.windows.net/MyEbms/@connectionParameters('CompanyId')/OData",
+          "x-ms-apimTemplate-operationName": [
+            "CreateProduct",
+            "UpdateProduct",
+            "GetProductCategoryID",
+            "CreateTask",
+            "UpdateTask",
+            "GetCustomerId",
+            "CreateCustomer",
+            "UpdateCustomer",
+            "CreateContact",
+            "GetCustomerCategoryId",
+            "CreateSalesOrder",
+            "GetTaskType",
+            "DeleteWebhook",
+            "GetTaskManager",
+            "TaskCreated",
+            "CustomerCreated",
+            "ProductCreated",
+            "SalesOrderCreated",
+            "CreateExpenseInvoice"
+          ]
+        }
+      },
+      {
+        "templateId": "setheader",
+        "title": "SerialNumber",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "serialnumber",
+          "x-ms-apimTemplateParameter.value": "@connectionParameters('SerialNumber','')",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      }
+    ],
+    "publisher": "EBMS AI",
+    "stackOwner": "Eagle Business Software"
```

Comment:
@bradleymarshall1982 for all our verified publisher we ask for the "publisher" and "stackOwner" properties to be the same. Thank you so much.

Reply:
Fixed.